contracts/TopShot.cdc

```diff
@@ -1094,25 +1057,52 @@ pub contract TopShot: NonFungibleToken {
     // Collection is a resource that every user who owns NFTs 
     // will store in their account to manage their NFTS
     //
-    pub resource Collection: MomentCollectionPublic, NonFungibleToken.Provider, NonFungibleToken.Receiver, NonFungibleToken.CollectionPublic, MetadataViews.ResolverCollection { 
+    access(all) resource Collection: MomentCollectionPublic, NonFungibleToken.Collection {
         // Dictionary of Moment conforming tokens
         // NFT is a resource type with a UInt64 ID field
-        pub var ownedNFTs: @{UInt64: NonFungibleToken.NFT}
+        access(contract) var ownedNFTs: @{UInt64: {NonFungibleToken.NFT}}
```

Comment:
```suggestion
        access(all) var ownedNFTs: @{UInt64: {NonFungibleToken.NFT}}
```
This needs to be `access(all)` now based on recent FLIPs and standard changes

Reply:
@joshuahannan with `access(all) var ownedNFTs: @{UInt64: {NonFungibleToken.NFT}}`, wouldn't it mean that anyone that gets a reference to an user's collection could move the NFTs out of it?

Reply:
no, you can't modify public dictionary or array fields.